src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/CommandLineApplication.cs

```diff
@@ -332,6 +332,10 @@ public void ShowHelp(string commandName = null)
             else
             {
                 target = Commands.SingleOrDefault(cmd => string.Equals(cmd.Name, commandName, StringComparison.OrdinalIgnoreCase));
+                if (target == null)
+                {
+                    throw new Exception(string.Format("TODO: Error: unexpected value '{0}' for option 'help'", commandName));
```

Comment:
Don't throw `Exception`. This should be either `ArgumentException` or `InvalidOperationException`